mu-plugins/blocks/site-breadcrumbs/src/view.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * Calculates the available space in the parent container between the breadcrumb container and the menu container.
+ *
+ * @param {HTMLElement} parent The parent container.
+ *
+ * @return {number} The available space in pixels
+ */
+const getAvailableSpace = ( parent ) => {
+	const parentWidth = parent.getBoundingClientRect().width;
+	let occupiedSpace = 0;
+
+	for ( const child of parent.children ) {
+		occupiedSpace += child.getBoundingClientRect().width;
+	}
```

Comment:
Could use reduce for this
```suggestion
	const occupiedSpace = parent.children.reduce( ( width, child ) => {
		return width + child.getBoundingClientRect().width;
	}, 0 );
```

Reply:
I definitely like `.reduce` when a functions are passed in. For simple operations like this, I don't feel strongly.

Reply:
IMO it's almost always the most elegant way to take many things (`children`) and output 1 thing (`width`); no mutation of the `let`, everything encapsulated in the callback.

But I don't feel strongly either 😂